### PR TITLE
browser: add browser based test

### PR DESCRIPTION
**Pull request by mmatczuk**
This test allows to generate real load coming from a browser. At the moment it only supports FireFox but can be easily extended with other browsers.

### CODE REVIEW

**Comment by mmatczuk** [comment]
I move that and local to testing. The e2e tests I left in the root. It feels like testing is all the manual inspection tools should go.

File: browser/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+export PATH  := $(PWD)/venv/bin:$(PATH)
+
+.PHONY: install
+install: venv
+	pip install -r requirements.txt
+	playwright install firefox
+
+.PHONY: test
+test:
+	@python ./topgun.py
```
Comment (Choraden) [inline browser/Makefile]: Let's unify python commands as test target uses `python` and venv `python3`.

**Comment by Choraden** [comment]
@mmatczuk this is super cool! 

Unfortunately, the script won't end for me. All browsers are long gone, forwarder does not have any open connections, yet the process lives on: 
```
make                                                                                    
pip install -r requirements.txt
Requirement already satisfied: greenlet==3.0.0 in ./venv/lib/python3.11/site-packages (from -r requirements.txt (line 1)) (3.0.0)
Requirement already satisfied: playwright==1.39.0 in ./venv/lib/python3.11/site-packages (from -r requirements.txt (line 2)) (1.39.0)
Requirement already satisfied: pyee==11.0.1 in ./venv/lib/python3.11/site-packages (from -r requirements.txt (line 3)) (11.0.1)
Requirement already satisfied: typing_extensions==4.8.0 in ./venv/lib/python3.11/site-packages (from -r requirements.txt (line 4)) (4.8.0)
playwright install Firefox

```

**Comment by mmatczuk** [comment]
It turns its silently fails on exception and hangs.
I added bunch of improvements, PTAL @Choraden.

**Comment by mmatczuk** [comment]
Nah it's still not good, I think I'd rewrite it in sync mode.

**Comment by Choraden** [comment]
Foxnews and wsj time out waiting, however in the browser it looks ok. I don't know if this has something to do with forwarder or the testing tool itself. I don't see any disturbing logs in forwarder though.

**Comment by Choraden** [comment]
LGTM, good job @mmatczuk

**Comment by mmatczuk** [comment]
Maybe they should be "talkative"

**Comment by Choraden** [comment]
> Maybe they should be "talkative"

It did help me with wsj. Foxnews still times out.

**Pull request merged**